crates/http/src/lib.rs

```diff
@@ -1,7 +1,19 @@
+use http_body_util::{combinators::BoxBody, BodyExt, Empty, Full};
+use hyper::body::Bytes;
+pub use wasmtime_wasi_http::body::HyperIncomingBody as Body;
+
 pub mod app_info;
 pub mod config;
 pub mod routes;
 pub mod trigger;
 pub mod wagi;
 
 pub const WELL_KNOWN_PREFIX: &str = "/.well-known/spin/";
+
+pub fn full(bytes: Bytes) -> Body {
+    BoxBody::new(Full::new(bytes).map_err(|_| unreachable!()))
+}
+
+pub fn empty() -> Body {
+    BoxBody::new(Empty::new().map_err(|_| unreachable!()))
+}
```

Comment:
Nit: we should probably put these in a `body` module so that it reads more nicely at the call site: e.g. `body::empty()`